cli/e2e.js

```diff
@@ -1,199 +1,10 @@
 /*jslint node: true */
 'use strict';
 
-const fs = require('fs-extra');
-const glob = require('glob');
-const path = require('path');
-const spawn = require('cross-spawn');
-const selenium = require('selenium-standalone');
-const protractorLauncher = require('protractor/built/launcher');
-const logger = require('@blackbaud/skyux-logger');
+const { start } = require('./utils/start-e2e');
```

Comment:
Now that `visual` is going to be refactored in to a library + plugin vs a command, is there any need for this refactor?

Reply:
Not necessarily; I was thinking it *could* be useful in case we came back to it with a different need; but I can pull it out.